test/Psy/Test/ConfigurationTest.php

```diff
@@ -132,6 +141,26 @@ public function testLoadConfigFile()
         $this->assertEquals(E_ALL & ~E_NOTICE, $config->errorLoggingLevel());
     }
 
+    public function testLoadLocalConfigFile()
+    {
+        $oldPwd = getenv('PWD');
+        putenv('PWD=' . realpath(__DIR__ . '/../../fixtures/default/'));
```

Comment:
@bobthecow maybe I should do all of the local file testing here instead of having those modifications in `testFilesAndDirectories`...?